tictactoe-app/src/test/java/camp/nextstep/edu/tictactoe/viewmodel/TictactoeViewModelTest.kt

```diff
@@ -13,26 +15,54 @@ class TictactoeViewModelTest {
     val rule = InstantTaskExecutorRule()
 
     private lateinit var viewModel: TictactoeViewModel
-    private lateinit var tictactoeGame: TictactoeGame
 
     @Before
     fun setUp() {
-        tictactoeGame = TictactoeGame()
-        viewModel = TictactoeViewModel(tictactoeGame)
+        viewModel = TictactoeViewModel()
     }
 
     @Test
-    fun `한 곳 입력 후 상태를 체크해본다`() {
+    fun `한 곳 입력 후 상태 체크해보면 진행중으로 나온다`() {
+        // given
         viewModel.clickCell(CellPosition.TOP_LEFT)
-        val actual = viewModel.uiState.value
-        assertThat(actual).isInstanceOf(GameResult.Status::class.java)
+        val result = viewModel.uiState.value
+        // when
+        if (result is GameResultUiState.Status) {
+            val actual = result.status
+            // then
+            assertThat(actual).isEqualTo(TictactoeStatus.Progress)
+        } else {
+            assertThat(result).isInstanceOf(GameResultUiState.Fail::class.java)
+        }
```

Comment:
else 문을 타는 경우에는 테스트 실패가 아닐까요?
```suggestion
        // then
        assertThat(result).isInstanceOf(GameResultUiState.Status::class.java)
        val actual = (result as? GameResultUiState.Status)?.status
        assertThat(actual).isEqualTo(TictactoeStatus.Progress)
```

Reply:
코드 보면서 한 수 배웠습니다.